src/main/java/uk/co/fivium/els/service/TemplatePopulator.java

```diff
@@ -56,7 +56,9 @@ public TemplatePopulator setRatingArrow(String parentElementId, RatingClass rati
     svgGroupElement.attr("transform", String.format("translate(0,%s)", yAxisTransform));
 
     setText(parentElementId+"Letter", rating.getLetter());
-    setText(parentElementId+"Plusses", rating.getPlusses());
+    if (!rating.getPlusses().isEmpty()) {
```

Comment:
Better to use `StringUtils.isNotBlank(rating.getPlusses())` as this also does null checking, which `isEmpty` doesn't. Its also slightly more semantic imo.

Reply:
Ah actually bigger problem is that this then doesn't _remove_ the plusses in the template if theyre are set. (generate a cabinet with a F rating for example).
This could check if the plusses element exists in the dom, although if they don't and the rating does have plusses doesn't that mean that the template is wrong or an invalid rating has been passed in, in which case we should just error anyway?